src/hypertrace/agent/init/__init__.py

```diff
@@ -152,9 +152,9 @@ def _init_exporter(self, trace_reporter_type):
         try:
             if trace_reporter_type == config_pb2.TraceReporterType.ZIPKIN:
                 exporter_type = 'zipkin'
-                exporter = ZipkinExporter(
-                    endpoint=self._config.agent_config.reporting.endpoint
-                )
+                #exporter = ZipkinExporter(
```

Comment:
will this give some form of npe? Maybe we can make this case point to noopexporter?

Reply:
no npe here. We assign exporter to None by default & then the caller of this func checks: 
```
        exporter = self._init_exporter(reporter_type)
        if exporter is None:
            logger.warning("Unable to initialize exporter")
            return
```